lib/defaults.js

```diff
@@ -2,260 +2,111 @@
-    'no-shadow': [2, {               // http://eslint.org/docs/rules/no-shadow
+    'no-shadow': [2, {               
       allow: ['res', 'req', 'err', 'next', 'resolve', 'reject', 'done', 'cb']
     }],
-    'no-unused-vars': [2, {          // http://eslint.org/docs/rules/no-unused-vars
+    'no-unused-vars': [2, {          
       'vars': 'all',
       'args': 'none'
     }],
-    'no-use-before-define': [2, 'nofunc'],       // http://eslint.org/docs/rules/no-use-before-define
+    'no-use-before-define': [2, 'nofunc'],       
 
     /**
      * Possible errors
      */
-    'comma-dangle': [2, 'only-multiline'],    // http://eslint.org/docs/rules/comma-dangle  @TODO: Turn into separate for front/back in dev env
-    'no-cond-assign': [2, 'except-parens'], // http://eslint.org/docs/rules/no-cond-assign
-    //'no-console': 1,                 // http://eslint.org/docs/rules/no-console
-    'no-debugger': 1,                // http://eslint.org/docs/rules/no-debugger
-    'no-alert': 2,                   // http://eslint.org/docs/rules/no-alert
-    //'no-constant-condition': 1,      // http://eslint.org/docs/rules/no-constant-condition
-    //'no-dupe-keys': 2,               // http://eslint.org/docs/rules/no-dupe-keys
-    //'no-duplicate-case': 2,          // http://eslint.org/docs/rules/no-duplicate-case
-    //'no-empty': 2,                   // http://eslint.org/docs/rules/no-empty
-    //'no-ex-assign': 2,               // http://eslint.org/docs/rules/no-ex-assign
-    'no-extra-boolean-cast': 2,      // http://eslint.org/docs/rules/no-extra-boolean-cast
-    'no-extra-parens': [2, 'functions'], // http://eslint.org/docs/rules/no-extra-parens
-    //'no-extra-semi': 2,              // http://eslint.org/docs/rules/no-extra-semi
-    //'no-func-assign': 2,             // http://eslint.org/docs/rules/no-func-assign
-    //'no-inner-declarations': 2,      // http://eslint.org/docs/rules/no-inner-declarations
-    //'no-invalid-regexp': 2,          // http://eslint.org/docs/rules/no-invalid-regexp
-    //'no-irregular-whitespace': 2,    // http://eslint.org/docs/rules/no-irregular-whitespace
-    //'no-obj-calls': 2,               // http://eslint.org/docs/rules/no-obj-calls
-    //'no-sparse-arrays': 2,           // http://eslint.org/docs/rules/no-sparse-arrays
-    //'no-unreachable': 2,             // http://eslint.org/docs/rules/no-unreachable
-    //'use-isnan': 2,                  // http://eslint.org/docs/rules/use-isnan
-    'valid-jsdoc': [2, {requireReturn: false}],
+    'comma-dangle': [2, 'only-multiline'],    
+    'no-cond-assign': [2, 'except-parens'], 
+    'no-debugger': 1,                
+    'no-alert': 2,                   
+    'no-extra-boolean-cast': 2,      
+    'no-extra-parens': [2, 'functions'], 
+    'valid-jsdoc': [1, {requireReturn: false}],
 
     /**
      * Best practices
      */
-    'consistent-return': [1, { treatUndefinedAsUnspecified: true}],          // http://eslint.org/docs/rules/consistent-return
```

Comment:
So there was a reason for all of these commented out rules, IIRC, it helps us easily check what's in airbnb vs what's in ours. It also makes it easy to flip it back to how it was in case airbnb changes their default.